tests/unit/client.test.ts

```diff
@@ -0,0 +1,102 @@
+import { AptosApiType } from "../../src/utils/const.js";
```

Comment:
lets have these tests part of this file https://github.com/aptos-labs/aptos-ts-sdk/blob/main/tests/e2e/client/aptosRequest.test.ts and a test for each AptosApiType

Reply:
this is a unit test and hence doesn't belong to aptosRequest.test.ts